### DISCUSSION TITLE
Change of colour scheme of the colour bar

### DISCUSSION THREAD

**Discussion by sajutabraham** [neperfepx/neper#530] [Q&A] [answered]
@rquey I would like to change the colour scheme of the colour bar to enhance the visualization of deformed microstructure. I used the command "-datapointcolscheme heat" but did not work. Either the command is wrong or I have kept it in the wrong place or my approach is wrong. Can you please guide me? The command lines given are as follows: 

neper -V ../myfile.sim \
    -simstep 2 \
    -datanodecoo coo \
    -dataelt1drad 0.004 \
    -dataelt3dedgerad 0.0015 \
    -datanodecoofact 10 \
    -datapointcolscheme plasma \
    -dataelt3dcol stress33 \
    -dataeltscaletitle "Stress (z) (MPa)" \
    -dataeltscale 0:500 \
    -showelt1d all \
    -cameraangle 13.5 \
    -imagesize 1000:1000 \
    -print 1_s33_deform_simstep2

**Comment by rquey** [comment, marked as answer]
Use `-dataeltcolscheme`, since you are coloring elements.

**Reply by sajutabraham** [reply]
Thanks a lot!
